manifest/openshift4/coreos/ncp-openshift4.yaml

```diff
@@ -224,6 +224,16 @@ rules:
 
 
 
+
```

Comment:
We already added this with PR #265, why is your PR detecting the diff?

Reply:
Perhaps it's just a rebase issue as your local branch is missing a few commits which are in main

Reply:
Thanks a lot for reminder, after rebasing local main branch and I get the changes from main, pleas take a look again. @salv-orlando